Doc/library/socketserver.rst

```diff
@@ -634,7 +648,7 @@ The output of the example should look something like this:
    Received: Thread-4: Hello World 3
 
 
-The :class:`ForkingMixIn` class is used in the same way, except that the server
-will spawn a new process for each request.
+The :class:`ForkingMixIn` and :class:`ProcessingMixIn` classes can be used in
+the same way, except that the server will spawn a new process for each request.
 Available only on POSIX platforms that support :func:`~os.fork`.
```

Comment:
Presumably `ProcessingMixIn` can be used on Windows, since `multiprocessing.Process` can

Reply:
Correct, that was my intent.

Since the multiprocessing.Process object is more portable, it makes sense to potentially use that if a developer wanted to.  I didn't think removing the existing ForkingMixIn class would be productive since that would break anything that uses it, and there is probably still good use cases to use a lighter weight "os.fork()" implementation on POSIX systems.

Reply:
What I had meant to say there was that the documentation was incorrect by implying that ProcessingMixIn was posix-only

Reply:
I think I understand now, it wasn't necessarily documentation that I changed but more documentation that I didn't change.  I'm updating the rst file now.